### ISSUE TITLE
result

### ISSUE THREAD

**Issue by shima97** [Khyat/fuzzy-washing#1]
hi, i have a fundamental  question about the number it shows as an output: the range is between 0-100 so the output should n't  be between 0-100? 
i mean a dirty(100) and fat(100) cloth needs to be washed for long time (and it should show 100 for that)
would you please explain the result more clearly?

**Comment by Khyat** [comment]
Sure, Thank you for showing your interest @shima97, actually the output isn't in the range of (0, 100) but it is in the range of (0, 61) it represents minutes in terms of how long it would take to wash the clothes according to the degree_of_dirtiness and type_of_dirtiness we specify when we input those parameters.

**Comment by shima97** [comment]
@Khyat thank you for your answer, ok but when we give 100 for degree_of_dirtiness and  100 for type_of_dirtiness it means cloths are very dirty and need the most time to be washed means 60 minutes.
but it shows 37.33 minutes as attached file shows.
i cant understand why? would you please explain that?
![Capture](https://user-images.githubusercontent.com/38297294/63835942-64997280-c92d-11e9-9e6a-83e5266c75f5.PNG)

**Comment by Haripriya378** [comment]
yeah i do have the same doubt .there is not much difference in the output. Could you please answer quickly?

**Comment by Vaibhav-Ankolekar** [comment]
## **_Solution :_**

Change `|` to `&` in rules in [fuzzify.py](https://github.com/Khyat/fuzzy_logic-washing_machine/blob/master/fuzzify.py) at line no. 26 to 34

<img width="538" alt="image" src="https://user-images.githubusercontent.com/76490924/188310321-91b994a5-3308-44dc-aedb-a8d4fb2768c2.png">

This will give output 

<img width="517" alt="image" src="https://user-images.githubusercontent.com/76490924/188310144-71c99361-81c2-496c-8368-424ae15d79fb.png">

Next, add defuzzification technique `'mom'` or `'som'` in consequent declaration at line no. 9 in [fuzzify.py](https://github.com/Khyat/fuzzy_logic-washing_machine/blob/master/fuzzify.py)
* `'centroid'` : Centroid of area
* `'bisector'` : bisector of area
* `'mom'`     : mean of maximum
* `'som'`       : min of maximum
* `'lom'`       : max of maximum

<img width="467" alt="image" src="https://user-images.githubusercontent.com/76490924/188324864-de45aa3f-abef-434b-ba53-b142c4d45ebe.png">

This will give the correct output

<img width="491" alt="image" src="https://user-images.githubusercontent.com/76490924/188325021-b3924c80-4462-448f-bdb4-a55bf07d37de.png">
